src/Modules/Build.lua

```diff
@@ -318,6 +318,7 @@ function buildMode:Init(dbFileName, buildName, buildXML, convertBuild)
 		{ stat = "AreaOfEffectRadius", label = "AoE Radius", fmt = "d" },
 		{ stat = "BrandAttachmentRange", label = "Attachment Range", fmt = "d", flag = "brand" },
 		{ stat = "BrandTicks", label = "Activations per Brand", fmt = "d", flag = "brand" },
+		{ stat = "Trauma", label = "Trauma", fmt = "d" },
```

Comment:
I feel like something like "Sustainable Trauma" would be a better label here. Maybe also add a bit of color to make it stand out a bit.

```LUA
 { stat = "Trauma", label = "Sustainable Trauma", fmt = "d", color = colorCodes.PHYS },
```

Reply:
![image](https://user-images.githubusercontent.com/31533893/188879803-2f29d440-f26c-4657-be11-3fd24e93bd2f.png)